### PR TITLE
ps

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/module/math/regular/pow.hpp [hunk 1]
```diff
@@ -106,21 +110,11 @@ namespace eve
 //!       * pow(\f$+\infty\f$, y) returns +0 for any y less than 0
 //!       * pow(\f$+\infty\f$, y) returns \f$+\infty\f$ for any y greater than 0
 //!       * except where specified above, if any argument is NaN, NaN is returned
+//!     2. [The operation is performed conditionnaly](@ref conditional)
+//!     3. speedier must less accurate call
```
Comment (jfalcou) [inline include/eve/module/math/regular/pow.hpp:114]: Faster is the correct word

File: include/eve/module/math/regular/pow1p.hpp [hunk 2]
```diff
@@ -38,45 +38,44 @@ namespace eve
...
+//!     * `m`: [Logical value](@ref logical) masking the operation.
 //!
 //! **Return value**
 //!
-//! Returns the [elementwise](@ref glossary_elementwise) \f$(1+x)^y\f$, with good accuracy,
-//! even when `x` is  small.
+//!     1. Returns the [elementwise](@ref glossary_elementwise) \f$(1+x)^y\f$, with good accuracy,
+//!        even when `x` is  small.
+//!     2. [The operation is performed conditionnaly](@ref conditional)
+//!     3. speedier must less accurate call.
```
Comment (jfalcou) [inline include/eve/module/math/regular/pow1p.hpp:75]: Faster

File: include/eve/module/math/regular/pow_abs.hpp [hunk 3]
```diff
@@ -47,53 +47,53 @@ namespace eve
...
+//!      *  pow_abs(x, \f$\pm0\f$) returns 1 for any x, even when x is NaN.
+//!      *  pow_abs(x, \f$-\infty\f$) returns \f$+\infty\f$ for any |x| `<` 1.
+//!      *  pow_abs(x, \f$-\infty\f$) returns \f$+0\f$ for any |x| `>` 1.
+//!      *  pow_abs(x, \f$+\infty\f$) returns \f$+0\f$ for any |x| `<` 1.
+//!      *  pow_abs(x, \f$+\infty\f$) returns \f$+\infty\f$ for any |x| `>` 1.
+//!      *  pow_abs(\f$+\infty\f$, y) returns \f$+0\f$ for any negative y.
+//!      *  pow_abs(\f$+\infty\f$, y) returns \f$+\infty\f$ for any positive y.
+//!      *  except where specified above, if any argument is NaN, NaN is returned.
+//!     2. [The operation is performed conditionnaly](@ref conditional)
+//!     3. speedier must less accurate call.
```
Comment (jfalcou) [inline include/eve/module/math/regular/pow_abs.hpp:93]: Faster

File: include/eve/module/math/regular/powm1.hpp [hunk 4]
```diff
@@ -48,34 +48,33 @@ namespace eve
...
+//!     * `m`: [Logical value](@ref logical) masking the operation.
 //!
 //! **Return value**
 //!
-//! Returns the [elementwise](@ref glossary_elementwise) power minus one, with good accuracy,
-//! even when `y` is very small, or when `x` is close to 1.
+//!    1. Returns the [elementwise](@ref glossary_elementwise) power minus one, with good accuracy,
+//!       even when `y` is very small, or when `x` is close to 1.
+//!     2. [The operation is performed conditionnaly](@ref conditional)
+//!     3. speedier must less accurate call.
```
Comment (jfalcou) [inline include/eve/module/math/regular/powm1.hpp:74]: Faster

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: include/eve/module/math/regular/pow.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/math/regular/pow.hpp]: done

File: include/eve/module/math/regular/pow1p.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/module/math/regular/pow1p.hpp]: done

File: include/eve/module/math/regular/pow_abs.hpp (hunk 3 above)
Comment (jtlap) [reply include/eve/module/math/regular/pow_abs.hpp]: done

File: include/eve/module/math/regular/powm1.hpp (hunk 4 above)
Comment (jtlap) [reply include/eve/module/math/regular/powm1.hpp]: done

**Review by jfalcou — APPROVED** [review]

**Pull request merged**